src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/AuditLogRepository.php

```diff
@@ -36,7 +36,9 @@ class AuditLogRepository extends EntityRepository
         'Surfnet\Stepup\Identity\Event\YubikeySecondFactorBootstrappedEvent',
         'Surfnet\Stepup\Identity\Event\GssfPossessionProvenEvent',
         'Surfnet\Stepup\Identity\Event\PhonePossessionProvenEvent',
-        'Surfnet\Stepup\Identity\Event\YubikeyPossessionProvenEvent',
+        'Surfnet\Stepup\Identity\Event\YubikeyPossessionProvenAndVerifiedEvent',
```

Comment:
Do you know why u2f events are not logged?

Reply:
Discussed IRL, can be added in a new story if need be.

Reply:
Checking with Pieter, I'll create a new story if it's unintentional.